developer/graphiql.html

```diff
@@ -0,0 +1,167 @@
+<!--
+ *  Copyright (c) 2021 GraphQL Contributors
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
```

Comment:
Ow, a copyright can of worms.

Please change this notice to:

This code is Copyright (c) 2021 GraphQL Contributors
<URL such and such>
reproduced here with permission from <URL to *their* LICENSE file>